Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.h

```diff
@@ -314,13 +316,12 @@ class RemoteImageBufferProxy : public WebCore::ConcreteImageBuffer<BackendType>
         if (UNLIKELY(!m_remoteRenderingBackendProxy))
             return false;
 
-        if (!m_needsFlush)
+        if (!m_hasPendingUpdates)
             return hasPendingFlush();
```

Comment:
Should we avoid this naming mix?  Using both m_hasPendingUpdates and hasPendingFlush() is a little bit confusing. 

Either we use needsFlush, hasPendingFlush, flushContext(). Or we use needsUpdate(), hasPendingUpdates(), updateContext().